src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -138,6 +146,13 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
             window_service = YaLMWindowService(service)
         elif model_name == "nvidia/megatron-gpt2":
             window_service = MegatronWindowService(service)
+        elif model_name in [
+            "together/llama-7b",
+            "together/alpaca-7b",
+            "together/vicuna-13b",
+            "together/redpajama-incite-base-3b-v1",
```

Comment:
RedPajama should be based on Pythia, not LLaMA, no?

Reply:
RedPajama, Pythia and MPT all use the GPT-NeoX tokenizer.

Reply:
Fixed this.

Reply:
Thank you!